docs/make.jl

```diff
@@ -15,6 +15,7 @@ makedocs(
         "Types" => "types.md",
         "Substitution" => "substitution.md",
         "Differentiation" => "differentiation.md",
+        "Antidifferentiation" => "antidifferentiation.md",
```

Comment:
Remove this line

Reply:
indeed, since `antidifferentiation` doc has been included in `differentiation.md`